taxcalc/tests/test_calculate.py

```diff
@@ -89,133 +69,130 @@ def test_sequence():
     run()
 
 
-# Create a basic Records object using Public Use File
-puf = Records(tax_dta)
-
-
 def test_make_Calculator():
-    # Create a Params object
-    params = Parameters(start_year=1991, inflation_rates=irates)
-    calc = Calculator(params, puf)
+    parm = Parameters()
+    assert parm.current_year == 2013
+    recs = Records(data=TAX_DTA, weights=WEIGHTS, start_year=2009)
+    calc = Calculator(params=parm, records=recs)
+    assert calc.current_year == 2013
 
 
 def test_make_Calculator_deepcopy():
     import copy
-    # Create a Params object
-    params = Parameters(start_year=1991, inflation_rates=irates)
-    calc = Calculator(params, puf)
-    calc2 = copy.deepcopy(calc)
+    parm = Parameters()
+    recs = Records(data=TAX_DTA, weights=WEIGHTS, start_year=2009)
+    calc1 = Calculator(params=parm, records=recs)
+    calc2 = copy.deepcopy(calc1)
 
 
 def test_make_Calculator_files_to_ctor(paramsfile):
     with open(paramsfile.name) as pfile:
         params = json.load(pfile)
     ppo = Parameters(parameter_dict=params, start_year=1991,
-                     num_years=len(irates), inflation_rates=irates)
-    calc = Calculator(params=ppo, records=tax_dta_path,
-                      start_year=1991, inflation_rates=irates)
+                     num_years=len(IRATES), inflation_rates=IRATES)
+    calc = Calculator(params=ppo, records=TAX_DTA_PATH,
+                      start_year=1991, inflation_rates=IRATES)
     assert calc
 
 
-def test_make_Calculator_mods():
-
-    # Create a Params object
-    params = Parameters(start_year=1991, inflation_rates=irates)
-
-    # Create a Public Use File object
-    puf = Records(tax_dta)
-
-    calc2 = calculator(params, puf, _II_em=np.array([4000]), _II_em_cpi=False)
+def test_make_Calculator_with_policy_reform1():
```

Comment:
This replacement test for `test_make_Calculator_mods` is redundant because the purpose of the original test was to test the ability to pass in particular reforms as keyword arguments. Again, I think @MattHJensen should chime in here to see if we should note this feature and create an issue for it to be included as a use case in creating a `Calculator` instance.

This new test exercises the same features as `test_make_Calculator_with_policy_reform2`, so I would suggest killing off `test_make_Calculator_with_policy_reform1` and leave  `test_make_Calculator_with_policy_reform2` and just call it  `test_make_Calculator_with_policy_reform`